docs/install.md

```diff
@@ -3,6 +3,7 @@ You have to set up several things before you can run a local instance of OAA for
 
 ## System Dependencies
 Start by installing these programs:
+ * Windows
```

Comment:
Added that since it took me an hour to realize that the dev tools are windows only.